spacifier.js

```diff
@@ -0,0 +1,8 @@
+const spacify = (str) => {
+	let splitResult = str.split('..');
+	if (splitResult.length === 1) return splitResult[0].split('').join(' ');
```

Comment:
maybe u can turn `string.split('').join(' ')` in a function as it's used twice

Reply:
Changed because I also like them spacez